core_domain.process_model_execution/src/flow_node_handler/boundary_event_handler/timer_boundary_event_handler.ts

```diff
@@ -27,10 +29,10 @@ export class TimerBoundaryEventHandler extends BoundaryEventHandler {
   private timerSubscription: Subscription;
 
   constructor(
-    eventAggregator: IEventAggregator,
-    flowNodePersistenceFacade: FlowNodePersistenceFacade,
-    timerFacade: ITimerFacade,
-    boundaryEventModel: Model.Events.BoundaryEvent,
+  @inject('EventAggregator') eventAggregator: IEventAggregator,
+    @inject('FlowNodePersistenceFacade') flowNodePersistenceFacade: FlowNodePersistenceFacade,
+    @inject('TimerFacade') timerFacade: ITimerFacade,
+    @unmanaged() boundaryEventModel: Model.Events.BoundaryEvent,
```

Comment:
Leftover? 
Bei den anderen Handlern sind keine Decorator vorangestellt.

```suggestion
    eventAggregator: IEventAggregator,
    flowNodePersistenceFacade: FlowNodePersistenceFacade,
    timerFacade: ITimerFacade,
    boundaryEventModel: Model.Events.BoundaryEvent,
```